pkg/sr/api.go

```diff
@@ -348,42 +348,59 @@ func (cl *Client) Schemas(ctx context.Context, subject string) ([]SubjectSchema,
 	return schemas, firstErr
 }
 
-// CreateSchema attempts to create a schema in the given subject.
+// CreateSchemaID attempts to create a schema with a fixed ID and version ID in the given subject,
+// and returns the globally unique identifier assigned by the registry. If the id is set to -1 or 0,
+// the registry will try to allocate a new identifier or reuse an existing one (see below).
+// If the versionID is set to -1 or 0, it will be omitted when creating the schema.
 //
-// This supports param [Normalize].
-func (cl *Client) CreateSchema(ctx context.Context, subject string, s Schema) (SubjectSchema, error) {
-	return cl.CreateSchemaWithIDAndVersion(ctx, subject, s, -1, -1)
-}
-
-// CreateSchemaWithIDAndVersion attempts to create a schema with a fixed ID and
-// version ID in the given subject. If the id is set to -1 or 0, this method is
-// equivalent to CreateSchema(). If the versionID is set to -1 or 0, it will be
-// omitted when creating the schema.
+// If the same schema is registered under a different subject, the same identifier will be returned.
```

Comment:
`the same identifier will be returned` -> `the original identifier will be returned` does this make more sense? These latter three lines could perhaps be a bit clearer. Also, isn't this line identical to the next?

```
If the schema is already registered, the original ID will be returned and a new version will not be created.
```

Does that cover it all?

Also, what happens if the schema does NOT exist but the id & version are already in use?

Reply:
> Also, what happens if the schema does NOT exist but the id & version are already in use?

Just tested this with our running Confluent Schema Registry 7.8.2 instance.
I tried to register a new dummy (but valid) schema under an existing schema ID and version number.
The Schema Registry server rejected the request with this error:
```
Subject mytopic-value is not in import mode
```

This seems to be part of [SR migrations](https://docs.confluent.io/platform/current/schema-registry/installation/migrate.html), which are valid use cases for this scenario.